cheeks.js

```diff
@@ -0,0 +1,48 @@
+const display = document.getElementById("display-info");
+const selector = document.querySelector("select");
+const button = document.querySelector("button");
+const reviewHolder = document.getElementById("reviews");
+const form = document.querySelector("form");
+const input = document.querySelector("input");
+const year = document.getElementById("release-year");
+const description = document.getElementById("description");
+const title = document.getElementById("title")
+
+const getTitles = async () => {
+  const allFilms = await axios.get("https://ghibliapi.herokuapp.com/films");
+  allFilms.data.forEach((film) => {
+    let option = document.createElement("option");
+    option.innerHTML = film.title;
+    selector.appendChild(option);
+  });
+};
+
+const submitReview = () => {
+  let review = document.createElement("li");
+  review.innerHTML = `<b>${selector.value}:</b> ${input.value}`;
+  reviewHolder.appendChild(review);
+};
+
+const showinfo = async () => {
+  const allFilms = await axios.get("https://ghibliapi.herokuapp.com/films");
```

Comment:
With the value property added to the option object we can make a call to a single film. The api call here should be to a [single film using its id](https://ghibliapi.herokuapp.com/#tag/Films%2Fpaths%2F~1films~1%7Bid%7D%2Fget).
 To get the id of the film that was chosen by the select dropdown we can utilize the `event` object ==> `e.target.value` or from the select variable `select.value`. The `value` property was used before to store the id of that movie. Once we get the data we just reassign the `innerText` for the title, release date, and description.